Assets/Scripts/World/LettersToReid/MyCharacterController.cs

```diff
@@ -11,7 +11,10 @@ public class MyCharacterController : MonoBehaviour {
     public bool isGrounded;
     public bool canClimb = false;
     
-
+	public Vector3[] memoryLocations = { new Vector3(5.76f, 0.26f, 0), new Vector3(16.21f, 2.37f, 0),
```

Comment:
I still don't think this is quite what we're going for. We shouldn't have to manually tell the system where the memories are. Instead, it should check (similar to the enemy) if the thing it collided with has the tag of "Letter"